playbooks/openstack-osh_hostconfig.yml

```diff
@@ -40,14 +42,15 @@
         autorefresh: True
         auto_import_keys: yes
         state: present
+        disable_gpg_check: yes
```

Comment:
Why is this disabled?

Reply:
Because the full image repo for SLE15-SP1 is not signed
No idea why, this should be temporary as we should move to use SCC for adding SLE repos instead of manually adding them

Reply:
Could you please add a comment there, to explain why we are doing this?

Something like
```#TODO(itxaka): Remove `disable_gpg_check` when SLE15SP1 image is signed```